### PR TITLE
pilar-portfolio

### PR DESCRIPTION
**Pull request by pilartorr**
I created a new folder with new files for this project.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  <meta charset="UTF-8">
+  <title>PILAR TORRES</title>
+  <link rel="stylesheet" href="styles/main.css">
+  <link rel="stylesheet" href="index.html">
+  <link rel="stylesheet" href="portfolio.html">
+  <link rel="stylesheet" href="contact.html">
+</head>
+
+<body>
+  <header class="content-header" style="padding:128px 16px">
```
Comment (itamargiv) [inline about.html]: `style="padding:128px 16px"` please try not to use the style attribute directly in your html files. If you want to debug something you can use the `element.style` rule in chrome dev tools.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>PILAR TORRES</title>
+  <link rel="stylesheet" href="styles/main.css">
+  <link rel="stylesheet" href="index.html">
```
Comment (itamargiv) [inline about.html]: Why are you adding all your other html pages as stylesheet links?

File: contact.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            <option value="australia">Spain</option>
+            <option value="canada">Germany</option>
+            <option value="usa">London</option>
+        </select>
+
+        <label for="subject">Subject</label>
+        <textarea id="subject" name="subject" placeholder="Write something.." style="height:200px"></textarea>
+
+        <input type="submit" value="Submit">
+      </form>
```
Comment (itamargiv) [inline contact.html]: Well done! :+1: This looks like a well structured form! Did you take it from somewhere else or write it yourself?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <title>PILAR TORRES</title>
+    <link rel="stylesheet" href="styles/main.css">
+    <link rel="stylesheet" href="about.html">
```
Comment (itamargiv) [inline index.html]: same comment as above

File: contact.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>PILAR TORRES</title>
+  <link rel="stylesheet" href="styles/main.css">
+  <link rel="stylesheet" href="index.html">
```
Comment (itamargiv) [inline contact.html]: Same as above

File: index.html [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+          <li><a href="portfolio.html">Portfolio</a></li>
+          <li><a href="contact.html">Contact</a></li>
+        </ul>
+      </nav>
+   </header>
+
+   <main class="main-content">
+      <h1>Pilar TORRES</h1>
+      <h2><button type="button">ENTER</button></h2>
+      <img src="" alt="background">
```
Comment (itamargiv) [inline index.html]: If you don't yet have an image, you can use placeimg.com for a placeholder.

File: contact.html [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+      <form>
+        <label for="fname">First Name</label>
+        <input type="text" id="fname" name="firstname" placeholder="Your name..">
+
+        <label for="lname">Last Name</label>
+        <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+
+        <label for="country">Country</label>
+        <select id="country" name="country">
+            <option value="australia">Spain</option>
```
Comment (itamargiv) [inline contact.html]: In select options, you need to make sure that the value matches somehow the text that is shown to the user. In this case, the value `australia` is not the text of this option, which is "Spain"

File: portfolio.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+      <ul class="navegation-list">
+        <li><a href="about.html">About</a></li>
+        <li><a href="portfolio.html">Portfolio</a></li>
+        <li><a href="contact.html">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <img src="#" alt="#">
```
Comment (itamargiv) [inline portfolio.html]: If you don't have images yet, you can use placeimg.com for placeholder images.

**Review by itamargiv — COMMENTED** [review]
Fantastic job pilar! :+1: From what I can see, the html matches the wireframes you showed me last week. You have a very good sense for structure. I did leave however a few comments, especially, I'm not sure why you added html files as stylesheet links.

File: styles/main.css [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+ input[type=submit]:hover {
+   background-color: #45a049;
+ }
+
+ .container {
+   border-radius: 5px;
+   background-color: #f2f2f2;
+   padding: 20px;
+ }
+</style>
```
Comment (itamargiv) [inline styles/main.css]: Please clean up your css file, we can't write html inside a `.css` file.

**Review by itamargiv — CHANGES_REQUESTED** [review]
Please correct the changes, and feel free to merge afterwards and work on your styles.

File: contact.html (hunk 3 above)
Comment (pilartorr) [reply contact.html]: I saw something like this in the orientation curs, but also I search it yesterday to remember how can I create a contact-box.

File: index.html (hunk 4 above)
Comment (pilartorr) [reply index.html]: I will delete this! Thanks

File: index.html (hunk 6 above)
Comment (pilartorr) [reply index.html]: Perfect! thanks!

**Pull request merged**